recipes/opentypespec/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "opentypespec" %}
+{% set version = "1.8.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opentypespec-{{ version }}.tar.gz
+  sha256: 03621635fed0d04345fa7b97837de31f992fa6619aeefcca0e75e342248dbde2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  # Windows cannot decode the 'charmap' codec byte 0x9d in position 721
+  skip: true  # [win]
```

Comment:
Sometimes that can be noarch and installation will be fine even if the build fails.

Reply:
@ocefpaf Sure - is it okay if I do this once the pull request is merged? I just want to make sure that all builds are working, as there are so many in this one pull request.